buildSrc/src/main/groovy/bio.terra.landingzone.java-common-conventions.gradle

```diff
@@ -47,18 +47,20 @@ dependencies {
     testImplementation 'ch.qos.logback:logback-classic'
     testImplementation 'org.hamcrest:hamcrest:2.2'
 
-    var openTelemetryVersion = '1.32.0'
+    var openTelemetryVersion = '1.34.1'
     implementation "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
     implementation "io.opentelemetry:opentelemetry-sdk:${openTelemetryVersion}"
     implementation "io.opentelemetry:opentelemetry-sdk-metrics:${openTelemetryVersion}"
     implementation "io.opentelemetry:opentelemetry-exporter-logging:${openTelemetryVersion}"
-    implementation "io.opentelemetry.semconv:opentelemetry-semconv:1.21.0-alpha"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-webmvc-6.0:${openTelemetryVersion}-alpha"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:${openTelemetryVersion}"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-boot:${openTelemetryVersion}-alpha"
+
+    var openTelemetryInstrumentationVersion = '2.0.0'
+    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-webmvc-6.0:${openTelemetryInstrumentationVersion}-alpha"
+    implementation "io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:${openTelemetryInstrumentationVersion}"
+    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-boot:${openTelemetryInstrumentationVersion}-alpha"
+
     implementation "com.google.cloud.opentelemetry:exporter-trace:0.25.2"
 
-    implementation ('bio.terra:terra-common-lib:0.1.6-SNAPSHOT'){
+    implementation ('bio.terra:terra-common-lib:1.0.9-SNAPSHOT') {
```

Comment:
@okotsopoulos Can you use 1.1.0 instead to pick up a security fix?

Reply:
It turns out that we would have gotten the needed security fix from 1.0.9 due to an earlier pin in TCL on jose4j, but I've taken the opportunity to upgrade to 1.1.0 instead, which addresses the dependency via the k8s client upgrade instead.  Thanks for flagging!

Reply:
Great, because there is yet ANOTHER vulernability that we get the fix for by upgrading to 1.1.0… bouncycastle. It's a never-ending battle.

https://github.com/DataBiosphere/terra-common-lib/pull/146#issuecomment-2004972839